sw/device/silicon_creator/lib/error.h

```diff
@@ -126,7 +127,9 @@ enum module_ {
   \
   X(kErrorSpiDevicePayloadOverflow,   ERROR_(1, kModuleSpiDevice, kInternal)), \
   \
-  X(kErrorAstInitNotDone,             ERROR_(1, kModuleAst, kInternal))
+  X(kErrorAstInitNotDone,             ERROR_(1, kModuleAst, kInternal)), \
+  \
+  X(kErrorRndBadCrc32,                ERROR_(5, KModuleRnd, kInvalidArgument))
```

Comment:
nit: shouldn't this start from 1?
```suggestion
  X(kErrorRndBadCrc32,                ERROR_(1, KModuleRnd, kInvalidArgument))
```

Reply:
Done